cos_registration_server/api/views.py

```diff
@@ -259,3 +266,110 @@ def delete(self, request: Request, uid: str) -> Response:
         dashboard = self._get_dashboard(uid)
         dashboard.delete()
         return Response(status=204)
+
+
+class PrometheusAlertRulesView(APIView):
+    """PrometheusAlertRules API view."""
+
+    def get(self, request: Request) -> Response:
+        """Prometheus Alert Rules get view.
+
+        request: Http GET request.
+        return: Http JSON response.
+        """
+        alert_rules = PrometheusAlertRule.objects.all()
```

Comment:
Don't you want here only the rules that are not template?

Reply:
I'm fetching them all so that I serialise both template and non-template

Reply:
But then from this value you don't do any operation and you serialized template that are not rendered.